tests/dotnet/UnitTests/TemplateTest.cs

```diff
@@ -1,28 +1,63 @@
 using System.Text.Json;
-using System.Text.Json.Serialization;
 
 namespace Xamarin.Tests {
+	public enum TemplateLanguage {
+		CSharp,
+		FSharp
+	}
+
+	public static class TemplateLanguageExtensions {
+		public static string AsString (this TemplateLanguage @this)
+		{
+			return @this switch {
+				TemplateLanguage.CSharp => "csharp",
+				TemplateLanguage.FSharp => "fsharp",
+				var language => throw new NotImplementedException ($"'{language} is not implemented.'")
```

Comment:
NBD, but this makes the code identical to the other extension methods:

```suggestion
				_ => throw new NotImplementedException ($"'{@this} is not implemented.'")
```